renderdoc/driver/vulkan/vk_postvs.cpp

```diff
@@ -56,6 +56,21 @@ static void ConvertToMeshOutputCompute(const ShaderReflection &refl, const SPIRV
 {
   SPIRVEditor editor(modSpirv);
 
+  // strip away any execution modes from the original
+  for (SPIRVIterator it = editor.BeginAll(); it; it++) {
```

Comment:
I don't think this new function is needed. The spec has a clear layout for where things are expected to happen, and `OpExecutionMode` opcodes only happen after entry points and before debug instructions.

I don't have each section precisely but you should be able to iterate from `BeginEntries` to `BeginDebug` and that will cover the area where execution modes could be declared.

Reply:
Oh!  I spent a while trying to figure out how I was supposed to access those sections, since there were not obvious helpers to fetch those areas, and there was no obvious way to loop everything.  Will fix that in a moment.